.circleci/config/jobs/push_operator_to_community_operators.yml

```diff
@@ -35,6 +35,14 @@ steps:
         NEW_OPERATOR_VERSION=${LATEST_TAG}
         echo "export NEW_OPERATOR_VERSION=${NEW_OPERATOR_VERSION}" >> $BASH_ENV
 
+  - run:
```

Comment:
You don't want to have a var for this command as we use it twice in the same way?

Reply:
That's a good point but it kind of depends on the variables in the previous steps.
I guess we could have 2 commands:
- export last released operator and last released snyk-monitor
- this step which ends tests early if nothing has changed

Not sure if it's not gonna become harder to track... instead of having everything in one file you now have to look up commands somewhere else! This is also very specific only to the Operators so I'm not sure it's making things more maintainable.